src/game_server/combat_update_packet.rs

```diff
@@ -0,0 +1,35 @@
+use byteorder::{LittleEndian, WriteBytesExt};
+use packet_serialize::{DeserializePacket, SerializePacket, SerializePacketError};
+use crate::game_server::game_packet::{GamePacket, OpCode};
+
+#[derive(Copy, Clone, Debug)]
+pub enum CombatUpdateOpCode {
+	ProcessedAttack        = 0x7

```

Comment:
nit: please use 4 spaces instead of tabs to be consistent with existing codebase